pkg/controllers/monitoring/monitoring-stack/controller.go

```diff
@@ -37,6 +37,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	stack "github.com/rhobs/observability-operator/pkg/apis/monitoring/v1alpha1"
+	tqctrl "github.com/rhobs/observability-operator/pkg/controllers/monitoring/thanos-querier"
```

Comment:
Importing `pkg/controllers/monitoring/thanos-querier` feels a bit weird to me, especially since Thanos Q probably doesn't need the version information. I'd decouple the 2 packages even if it means that code duplication.

Reply:
I used the thanos-querier type already in `pkg/operator/operator.go`. I'll add a commit where `monitoringstack` has its own type.